packages/toolkit/src/view/model/view-model/ModelContentViewer.tsx

```diff
@@ -23,6 +28,20 @@ export const ModelContentViewer = ({
   onUpdate,
   modelState,
 }: ModelContentViewerProps) => {
+  const router = useRouter();
+  const routeInfo = useRouteInfo();
+  React.useEffect(() => {
+    if (model) {
+      if (!model.permission.canEdit) {
+        const playgroundPath = `/${routeInfo.data?.namespaceId}/models/${model.id}/playground`;
+        router.push(playgroundPath);
+      } else {
+        console.log("User has permission to edit model");
+      }
+    } else {
+      console.log("Model data not available");
+    }
+  }, [model, routeInfo.data?.namespaceId, router]);
```

Comment:
@thewbuk Using console.log won't hint user they don't have permission to edit this model. We should hint them this issue

Reply:
Given they are immediately redirected to the playground, I would opt for simply removing the logs. Otherwise, the flow would need to be: user lands on model they cannot edit -> add some time out to show the notification -> redirect @EiffelFly

Reply:
@thewbuk Ah I think we only need to do this redirection in the setting tab. Version, Playground, Readme, API tab should be public available